pkg/ebpf/c/tracee.bpf.c

```diff
@@ -2539,6 +2846,28 @@ int tracepoint__raw_syscalls__sys_enter(struct bpf_raw_tracepoint_args *ctx)
         sys->id = *id_64;
     }
 
+    if (should_submit(sys->id, data.config) &&
+        has_syscall_fd_arg(sys->id) &&
+        data.config->options & OPT_CAPTURE_FD_FILEPATH) {
```

Comment:
I believe OPT_CAPTURE_FD_FILEPATH should be checked first, then should_submit, then has_syscall_fd_arg.

Reply:
Limiting the checks only for `option:parse-arguments-fds`, yeah, we can avoid a lot of processing reading OPT_CAPTURE_FD_FILEPATH first.